pavement.py

```diff
@@ -684,8 +684,9 @@ def write_release_task(options, filename='NOTES.txt'):
 """)
         ftarget.writelines(['%s\n' % c for c in compute_sha256(idirs)])
 
-    # Sign release
-    cmd = ['gpg', '--clearsign', '--armor']
+    # Sign release; on some platforms gpg2 may actually
+    # be named gpg
+    cmd = ['gpg2', '--clearsign', '--armor']
```

Comment:
Maybe make the choice between gpg/gpg2 configurable? I'm pretty sure it worked for me as it was, on a bog-standard ubuntu box.

Reply:
Usually those are aliases. I'd suggest to just merge this as is, and in case it doesn't work to just fix up this line. Rather than adding a switch, that's basically the same (first invocation doesn't work, oh I need to read `pavement.py` and look at what switch to use).

Reply:
Ha, I ran into this Monday after I had upgraded to fedora 30, which silently changed gpg to point to gpg2 and all my secret keys disappeared because gpg2 keeps them in the same file as the public keys instead of separately. It took me *hours* to figure out what was going on. Then I needed to install gpg1 so I could export the keys to gpg2 for import. So a heads up that there are side effects to making the change.

Reply:
But the change did make uploading much easier once I got things going.